src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -127,7 +127,7 @@ you can [develop new custom packages][].
 ## Managing package dependencies and versions
 
 To minimize the risk of version collisions,
-specify a version range in the pubspec file.
+specify a version range in the `pubspec.yaml` file.
```

Comment:
It looks like you've done a sweep through the site looking for "the pubspec file" and changing it to `pubspec.yaml`. It's fine, but seems like a little overkill.  :)

Reply:
If I had done a proper sweep, I would have touched 10 more occurrences in 8 more files :) 

It's just something I came across while fixing other issues. LMK if you'd like any of these to stay as pubspec file.

Reply:
I don't have a strong opinion. We definitely use both on the dart site.